### DISCUSSION TITLE
[Request] Support BrowserRouter

### DISCUSSION THREAD

**Discussion by banool** [community/community#44678] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As explained in this Stack Overflow post, GitHub Pages does not support the `BrowserRouter` approach for `react-router`: https://stackoverflow.com/questions/58228017/react-router-v4-cant-load-page-on-github-pages. In short, you either have to use `HashRouter` or a gross hack with `404.html`.

Could you please consider some way to make `BrowserRouter` work out of the box with GitHub Pages? This to me is one of the major usability problems with GH Pages today.

Thanks!

**Comment by mcaquet** [comment]
Agreed!  Have you found a solution to deploy your app on GitHub Pages ?

For info, I'm still looking for a solution for my React app:
https://github.com/orgs/community/discussions/51200
